src/types/secret.ts

```diff
@@ -19,24 +31,115 @@ type linkServiceAccount = {
 export type RemoteSecretKind = K8sResourceCommon & {
   spec: {
     secret: {
-      linkedTo: linkServiceAccount[];
+      linkedTo?: linkServiceAccount[];
       name: string;
       type: string;
+      labels?: {
+        [key: string]: string;
+      };
+      annotations?: {
+        [key: string]: string;
+      };
     };
     targets: { namespace: string }[];
   };
+  status?: RemoteSecretStatus;
 };
 
+export interface RemoteSecretStatus {
+  conditions?: Condition[];
+  secret?: Secret;
+  targets?: Target[];
+}
+
+export enum RemoteSecretStatusReason {
+  AwaitingData = 'AwaitingData',
+  DataFound = 'DataFound',
+  Injected = 'Injected',
+  PartiallyInjected = 'PartiallyInjected',
+  Error = 'Error',
+  Unknown = '-',
+}
+
+export enum RemoteSecretStatusType {
+  Deployed = 'Deployed',
+  DataObtained = 'DataObtained',
+}
+export interface Condition {
+  lastTransitionTime: string;
```

Comment:
Looks like this is causing build failure because there is already another export called `Condition`

Reply:
fixed it.